incident_viewer_demo/src/LineChart/LineChart.jsx

```diff
@@ -451,10 +514,10 @@ class LineChart extends PureComponent {
         return value;
       });
     }
-    console.log(annotationSeries)
-    console.log(highlightValues)
-    
-    this.updateIncident();
+
+    if (hasIncident) {
```

Comment:
I'm a little confused with following this hasIncident boolean around... Where is it set to true?